libuavcan/src/transport/uc_can_acceptance_filter_configurator.cpp

```diff
@@ -16,24 +16,22 @@ const unsigned CanAcceptanceFilterConfigurator::DefaultAnonMsgID;
 
 int16_t CanAcceptanceFilterConfigurator::loadInputConfiguration(AnonymousMessages load_mode)
 {
-    multiset_configs_.clear();
```

Comment:
Agree, so I will mention in tutorial and header that computeConfiguration() must be called before addFilterConfig().